options/locale/locale_en-US.ini

```diff
@@ -1946,8 +1946,8 @@ pulls.delete.title = Delete this pull request?
 pulls.delete.text = Do you really want to delete this pull request? (This will permanently remove all content. Consider closing it instead, if you intend to keep it archived)
 
 pulls.recently_pushed_new_branches = You pushed on branch <strong>%[1]s</strong> %[2]s
-pulls.upstream_diverging_prompt_behind_1 = This branch is %d commit behind %s
-pulls.upstream_diverging_prompt_behind_n = This branch is %d commits behind %s
+pulls.upstream_diverging_prompt_behind_1 = This branch is %[1]d commit behind %[2]s
```

Comment:
No modification is required here

Reply:
What we need to modify should be `options/locale/locale_zh-CN.ini`
```ini
pulls.upstream_diverging_prompt_behind_1=该分支落后于 %[2]s %[1]d 个提交
pulls.upstream_diverging_prompt_behind_n=该分支落后于 %[2]s %[1]d 个提交
```

Reply:
No, it can't be done on main branch. The backport needs to manually fix it.

Reply:
Maybe some maintainers should document how the Crowdin works @techknowlogick @lunny

Reply:
Sorry, I don't understand this workflow yet

Reply:
Yup, I do not quite understand either, so I mentioned techknowlogick & lunny

Reply:
Crowdin sync will only happen on the main branch. For other branches, we have to backport manually.